### ISSUE TITLE
[Inky] <center> tags need to add class and attribute to child elements

### ISSUE THREAD

**Issue by rafibomb** [foundation/foundation-emails#190]
Wrapping an element in a center tag should output the align attribute and class as well.

```
<center>
  <button href="http://zurb.com">Text</button>
</center>
```

Expected output

```
<center>
   <table class="button center" align="center">
```

These extra methods ensure cross client compatibility.

**Comment by gakimball** [comment]
We don't have a custom tag for `<center>`. @tdhartwick thoughts?

One consideration is what happens if you put multiple things inside of a `<center>` tag (I don't know if this is valid syntax or not):

``` html
<center>
  <button></button>
  <row></row>
</center>
```

Do we apply the class `.center` and the attribute `align="center"` to every sub-element?

**Comment by rafibomb** [comment]
This could be handled in the documentation. The `<center>` tag wrapped around an element should add the attributes to the one element it's wrapped around

or

someone can add the .center class and Inky will wrap the element in `<center>` tags and add the align attribute.

**Comment by gakimball** [comment]
Merged zurb/inky#11 and zurb/inky#12

Shipped in Inky 1.1.0.
